.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Bug Report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+INSTRUCTIONS
+==========
+
+If the below instructions are not followed, we may be forced to close your issue.
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the latest release to make sure your issues has not already been fixed: 
+https://github.com/Azure/azure-iot-sdk-c/releases/latest 
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoT Hub name, Device Provisioning Service scope ID), etc...). Please submit a ticket and get assistance from the Microsoft support team using the following link: 
+https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request
+
+4. Include enough information for us to address the bug, including a description of the bug, a Minimal Complete Reproducible Example that demonstrates the bug, and any relevant logs related to the bug.  
+
+For the MCVE, we recommend you provide us with a code snippet we can cut and paste into a readily available sample, or a link to a code sample that we can compile to reproduce the bug. 
```

Comment:
>MCVE [](start = 8, length = 4)

MCVE - please put the link to this concept [https://stackoverflow.com/help/mcve](https://stackoverflow.com/help/mcve) 1st time that you use it.

Reply:
Or else put an MCVE in parens and link ideally next to the Minimal... the 1st time you spell out the full concept in sentence above.

---
In reply to: [263177036](https://github.com/Azure/azure-iot-sdk-c/pull/886#discussion_r263177036) [](ancestors = 263177036)